lib/PlaidClient.js

```diff
@@ -205,6 +205,9 @@ Client.prototype.getAllTransactions =
             offset: transactionsCount,
           })
         );
+        if (!transactionsResponse.transactions) { // check if error was returned
```

Comment:
Could you instead do this?

```suggestion
        if (transactionsResponse instanceof Error) {
```